packages/grid/docker-compose.override.yml

```diff
@@ -166,8 +175,10 @@ services:
       dockerfile: headscale.dockerfile
     volumes:
       - headscale-data:/headscale/data
+      # - ./grid/vpn:/headscale
     environment:
       - NETWORK_NAME=omnet
+      - STACK_API_KEY=$STACK_API_KEY
```

Comment:
Containers which need to communicate via our internal container services will need this key in their ENV.